### DISCUSSION TITLE
How to send invoice amazon dba

### DISCUSSION THREAD

**Discussion by rsantosdosanjos** [abuzuhri/Amazon-SP-API-CSharp#591] [General]
How can I send invoices for amazon dba type orders

**Comment by zinsmeik** [comment]
If you want to uplaod the invoice as PDF see https://github.com/abuzuhri/Amazon-SP-API-CSharp/issues/546#issuecomment-1618288567

**Reply by rsantosdosanjos** [reply]
I need to send the invoice to this endpoint https://sellingpartnerapi-na.amazon.com/delivery/2022-07-01/invoice


POST /delivery/2022-07-01/invoice
https://pt-br-group.readme.io/amazon-english/docs/dba-use-case

**Reply by zinsmeik** [reply]
Sorry with this I can not help

**Reply by rsantosdosanjos** [reply]
@zinsmeik  Could you help me to know if the library provides the x-amz-access-token or some way to generate it?
